site/content/en/docs/start/_index.md

```diff
@@ -525,6 +525,11 @@ You can also make your life easier by adding the following to your shell config:
 ```shell
 alias kubectl="minikube kubectl --"
 ```
+for windows :
```

Comment:
Lets make this Automatic based on user's browser, same as download instructions that automatically goes

"https://minikube.sigs.k8s.io/docs/start/"